### ISSUE TITLE
GPU mode was launched when using Theano as backend server

### ISSUE THREAD

**Issue by huaiyanggongzi** [fchollet/keras#5732]
I have Keras installed with both Tensorflow and Theanon. I found that, when using Tensorflow as backend, the GPU mode can be launched; while using Theano as backend, the GPU mode is not launched. What are the possible reasons for this? Which kind of configuration should I modified/examined for this kind of problem? Thanks.

**Comment by nouiz** [comment]
Did you set Theano flag to enable the GPU? If you don't know what I talk
about, check in Theano doc how to enable the GPU. I'm offline so can't give
you the URL.

Le dim. 12 mars 2017 18:07, huaiyanggongzi <notifications@github.com> a
écrit :
